src/toast/toast.ts

```diff
@@ -118,7 +144,19 @@ export class NgbToast implements AfterContentInit,
 
   hide() {
```

Comment:
Could we add documentation for both `.hide()` and `.show()` methods? At the moment there is no documentation for this on the demo site?